src/loaders/express.js

```diff
@@ -23,33 +25,35 @@ module.exports = (app) => {
         },
     }));
 
-    // Applying middleware
     // JSON bodyparser (parses JSON request body into req.body)
     app.use(bodyParser.json());
 
-
+    // Initializing passport authentication settings
     app.use(passport.initialize());
     app.use(passport.session());
 
-    // Adding headers
-    app.use((req, res, next) => {
-    // Website to allow to connect
-        res.setHeader("Access-Control-Allow-Origin", "*");
+    // Adds route logging
+    app.use(morgan("common"));
 
-        // Request methods to allow
+    // Adding headers (CORS)
+    app.use((_, res, next) => {
+        // Allow connections for all origins
+        res.setHeader("Access-Control-Allow-Origin", "*");
+        // Allowed request methods
         res.setHeader("Access-Control-Allow-Methods", "GET, POST, OPTIONS, PUT, PATCH, DELETE");
```

Comment:
Are we using all of these? Shouldn't we remove the ones we don't use?

Reply:
This is CORS, which will be necessary for any API.

Reply:
If you are mentioning the allowed methods, this was probably just a left-over copy pasta from somewhere online for enabling CORS for node+express

Reply:
I was asking if all the http methods there will be used, if not, they can be removed from the `Access-Control-Allow-Methods` Header

Reply:
I don't really see a problem, as they would not match any endpoint and just return a 404. But I get what you mean.